bitrise.yml

```diff
@@ -162,7 +162,7 @@ workflows:
             set -euxo pipefail
             echo "$BITRISE_ARTIFACT_PATHS"
 
-            derived_data=${BITRISE_ARTIFACT_PATHS#*|}
+            derived_data=${BITRISE_ARTIFACT_PATHS##*|}
```

Comment:
what does this extra `#` give us here?

Reply:
The BITRISE_ARITIFACT_PATHS has three different files now (it also stores the swiftlint_report), it had only two before. They are separated by `|` , like:
BITRISE_ARITIFACT_PATHS=path/to/file1|path/to/file2|path/to/Derived_data

With one `#` we get the path after the first `|` with two `##` we get the path after the latest `|` which is what we want.